src/components/blog-list-client.tsx

```
@@ -205,101 +192,82 @@ export function BlogListClient({ posts }: BlogListClientProps) {
           {hasActiveFilters && (
-            <div className="relative z-50 flex items-center justify-between mb-6">
-              <p className="text-sm text-muted-foreground">
-                {filteredPosts.length}{" "}
-                {filteredPosts.length === 1 ? "post" : "posts"} found
-              </p>
+            <div className="flex justify-center mb-8">
               <button
                 onClick={handleClearAll}
-                className="text-sm text-muted-foreground hover:text-foreground transition-colors"
+                className="text-sm text-muted-foreground hover:text-foreground transition-colors border-b border-transparent hover:border-muted-foreground"
               >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)